search/tasks.py

```diff
@@ -0,0 +1,21 @@
+"""Indexing tasks"""
+from search import indexing_api as api
+from open_discussions.celery import app
+
+
+@app.task
+def create_document(doc_id, data):
+    """Task that makes a request to create an ES document"""
+    api.create_document(doc_id, data)
+
+
+@app.task
```

Comment:
To address my earlier concern about race conditions between this task and the `create_document` one, I think we should have this task backoff and retry (up to some limit) if a partial update fails because the document doesn't exist yet. That's probably the simplest / most elegant solution.

Same for `increment_document_integer_field` below.

Reply:
Yeah, any partial update request should be safely retry-able. Celery task retry setting seems like a sensible safeguard against the possibility of partial update tasks running before the document is created